src/pages/home/Home.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+import './Home.css'
+
+const Home = () => {
+  const navigate = useNavigate()
+  return (
+    <div className="home">
+      <button className="home-btn" onClick={() => navigate('/counter')}>
```

Comment:
For the internal navigation use link / href instead of button with onClick

Reply:
Used a tag for that.